src/js/common/components/Modal.jsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import classNames from 'classnames';
 
-function focusListener() {
+function focusListener(selector) {
   const listener = event => {
     const modal = document.querySelector('.va-modal');
     if (!modal.contains(event.target)) {
       event.stopPropagation();
-      modal.focus();
+      const focusableElement = document.querySelector(selector);
```

Comment:
Will the element always be inside the modal? Could you use `modal.querySelector(selector)` here to avoid searching the whole DOM?